app/actions/remote/scheduled_post.ts

```diff
@@ -49,16 +49,14 @@ export async function fetchScheduledPosts(serverUrl: string, teamId: string, inc
         }
 
         const scheduledPostsResponse = await client.getScheduledPostsForTeam(teamId, includeDirectChannels, groupLabel);
-        const {directChannels, ...scheduledPostsByTeam} = scheduledPostsResponse;
+        const {directChannels = [], ...scheduledPostsByTeam} = scheduledPostsResponse;
         const scheduledPosts = [...Object.values(scheduledPostsByTeam).flat(), ...directChannels];
-        if (scheduledPosts.length) {
```

Comment:
Why are we omitting this check now?

Reply:
It was wrong initially. We need to call the handler even when length is 0 to delete the scheduled posts.